common/djangoapps/student/role_helpers.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
this doesn't seem ideal, but if i tried putting this in student/roles or student/helpers I was getting a lot of circular import issues, so open to other suggestions